### PR TITLE
Update README.md: Error Handling & Deployment Experience Video

### PR DESCRIPTION
**Pull request by kaydaskalakis**
- Added "Installation Error Handling" for Windows Users to assist first timers or users migrating from BloodHound Legacy in deploying BloodHound CE.
- Added a video covering the deployment experience on Windows.

## Types of changes

-   [x] Chore (a change that does not modify the application functionality)
-   [ ] Bug fix (non-breaking change which fixes an issue)
-   [ ] New feature (non-breaking change which adds functionality)
-   [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

-   [x] Documentation updates are needed, and have been made accordingly.
-   [ ] I have added and/or updated tests to cover my changes.
-   [ ] All new and existing tests passed.
-   [ ] My changes include a database migration.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,14 +24,33 @@ original BloodHound was created by [@_wald0](https://www.twitter.com/_wald0), [@
 
 The easiest way to get up and running is to use our pre-configured Docker Compose setup. The following steps will get BloodHound CE up and running with the least amount of effort.
   
-  1. Install Docker Compose. This should be included with the [Docker Desktop](https://www.docker.com/products/docker-desktop/) installation
-  2. Run `curl -L https://ghst.ly/getbhce | docker compose -f - up`
-  3. Locate the randomly generated password in the terminal output of Docker Compose
-  4. In a browser, navigate to `http://localhost:8080/ui/login`. Login with a username of `admin` and the randomly generated password from the logs
+  1. Install Docker Compose and ensure Docker is running. This should be included with the [Docker Desktop](https://www.docker.com/products/docker-desktop/) installation
+  3. Run `curl -L https://ghst.ly/getbhce | docker compose -f - up`
```
Comment (StephenHinck) [inline README.md:28]: Please correct numbering

File: README.md [hunk 2]
```diff
@@ -24,14 +24,33 @@ original BloodHound was created by [@_wald0](https://www.twitter.com/_wald0), [@
...
+  5. In a browser, navigate to `http://localhost:8080/ui/login`. Login with a username of `admin` and the randomly generated password from the logs
 
 NOTE: going forward, the default `docker-compose.yml` example binds only to localhost (127.0.0.1). If you want to access BloodHound outside of localhost,
 you'll need to follow the instructions in [examples/docker-compose/README.md](examples/docker-compose/README.md) to configure the host binding for the container.
 
+## Installation Error Handling
+
+<i> Affects: Windows Users </i>
+
+- If you encounter a "failed to get console mode for stdin: The handle is invalid." ensure Docker Desktop (and associated Engine is running). Docker Desktop does not automatically register as a startup entry. <img width="302" alt="image" src="https://github.com/kaydaskalakis/BloodHoundCE/assets/8494920/4ff6f784-b60d-4b59-821d-d3d351f6df3b">
```
Comment (StephenHinck) [inline README.md:39]: Please include this image within the BH repo and link directly, not to an outside repo

File: README.md [hunk 3]
```diff
@@ -24,14 +24,33 @@ original BloodHound was created by [@_wald0](https://www.twitter.com/_wald0), [@
...
+```
+# Verify if Docker Engine is Running
+docker info
+
+# Attempt to stop Neo4j Service if running
+Stop-Service "Neo4j" -ErrorAction SilentlyContinue
+```
+- A succesfull installation of BloodHound CE would look like below:
+
+https://github.com/kaydaskalakis/BloodHoundCE/assets/8494920/537b346d-0451-4c39-898f-da5a57575272
```
Comment (StephenHinck) [inline README.md:52]: Please include this image within the BH repo and link directly, not to an outside repo

**Review by StephenHinck — DISMISSED** [review]
Looks great overall - couple minor nits and then happy to approve!

**Review by kaydaskalakis — COMMENTED** [review]
Changes in line with request.

**Review by rvazarkar — APPROVED** [review]

**Pull request merged**